router.go

```diff
@@ -349,7 +349,9 @@ func (app *App) registerStatic(prefix, root string, config ...Static) Router {
 		if maxAge > 0 {
 			cacheControlValue = "public, max-age=" + strconv.Itoa(maxAge)
 		}
-
+		if config[0].CacheDuration != 0 {
```

Comment:
what happens if someone wants to set it to 0 ? is the 0 value possible

Reply:
If you want to disable it, you need to provide a negative number i.e. `-1`
![](https://i.imgur.com/MV6Z5mz.png)

Reply:
okay, not needed 
https://github.com/valyala/fasthttp/blob/805af0ee73bd914c4ee3ee8eccf400edaa0ccfb7/fs.go#L369
0 is not possible in fs.go

green light for the PR